lib/mbta_v3_api/facilities/repo.ex

```diff
@@ -26,4 +26,21 @@ defmodule MBTAV3API.Facilities.Repo do
       facilities_filter_by_fn.([{"stop", stop_id}])
     end)
   end
+
+  def get_by_type(type, opts \\ [])
+
+  def get_by_type(type, opts) when is_list(type) do
+    type |> Enum.join(",") |> get_by_type(opts)
```

Comment:
If a list you passed into this is joined by `"` and then used for the filter on line 38, would the filter work properly if the order of the items was unique?

Reply:
Sorry, I'm not 100% sure what you're asking 😅 the API filter takes in types as a comma-delineated string, so this is just there to let users pass it in as a list instead if they want to. The order of the types in the filter doesn't matter.